Interesting to see that mypy actually reports an error on [travis](https://travis-ci.org/home-assistant/home-assistant/jobs/149305990):

```
homeassistant/helpers/typing.py:4: error: Module has no attribute 'NewType'
homeassistant/helpers/typing.py:6: error: Name 'Dict' already defined
homeassistant/helpers/typing.py:14: error: Generic type is prohibited as a runtime expression (use a type alias or '# type:' comment)
```

--- a/homeassistant/helpers/typing.py
+++ b/homeassistant/helpers/typing.py
@@ -1,18 +1,12 @@
 """Typing Helpers for Home-Assistant."""
-# pylint: disable=invalid-name
-try:
-    from typing import NewType, Dict, Any
-except ImportError:
-    from typing import Dict, Any
-
-    def NewType(name: str, typ: Any) -> Any:
-        """Fake NewType, required for Python 3.5.1."""
-        return typ
+from typing import Dict, Any
 
 import homeassistant.core
 
-ConfigType = NewType('ConfigType', Dict[str, Any])
+# pylint: disable=invalid-name
+
+ConfigType = Dict[str, Any]
 HomeAssistantType = homeassistant.core.HomeAssistant
 
 # Custom type for recorder Queries
-QueryType = NewType('QueryType', Any)
+QueryType = Any

